core/sphinx/kemsphinx_test.go

```diff
@@ -124,34 +127,34 @@ func newKEMPathVector(require *require.Assertions, mykem kem.Scheme, nrHops int,
 	}
 
 	// Assemble the path vector.
-	path := make([]*PathHop, nrHops)
-	for i := range path {
-		path[i] = new(PathHop)
-		copy(path[i].ID[:], nodes[i].id[:])
-		path[i].KEMPublicKey = nodes[i].publicKey
+	p := make([]*path.PathHop, nrHops)
```

Comment:
is the rename to `p` strictly necessary here ? it kind of looks like it's to avoid the clash with the module name `path`, which again hints to me that we maybe should find a different name for that module.